src/js/distrlbute.js

```diff
@@ -0,0 +1,59 @@
+function shuffle(arr) {
+  for (let i = arr.length - 1; i > 0; i--) {
+    let j = Math.floor(Math.random() * (i + 1)); //random index
+    [arr[i], arr[j]] = [arr[j], arr[i]]; // swap
+  }
+}
+
+/***
+ *
+ * 重みつけ振り分け処理
+ *
+ * @param nameList
+ * @param mailList
+ * @param listWeight
+ * @param groupNum
+ * @returns {*[]}
+ */
+function distributeGroupMember(nameList, mailList, listWeight, groupNum) {
+  // メールリストと名前リストをまとめる
+  const unitList = [];
+  for (let i = 0; i < nameList.length; i++) {
+    unitList.push({ name: nameList[i], mail: mailList[i] });
+  }
+
+  const groupList = [];
+  let weightNum = 0;
+  const tmpList = [];
+  const shuffledList = [];
```

Comment:
```suggestion
  const groupList = [];
  const tmpList = [];
  const shuffledList = [];
  let weightNum = 0;
  
  // メールリストと名前リストをまとめる
  const unitList = [];
  for (let i = 0; i < nameList.length; i++) {
    unitList.push({ name: nameList[i], mail: mailList[i] });
  }
```

const上でletが下、全体で使う変数は上の方にまとめたい

Reply:
対応しました